app/authentication/forms.py

```diff
@@ -10,19 +10,19 @@
 
 
 
-
```

Comment:
•	Lack of comment in the code. There are limited comments both in the front end and back end code. Anyone who reads the code can not figure out completely the scope of the code. In other words is not quite readable for someone who did not contribute to the code. I does not happen only in this code but in most of them